packages/compass-e2e-tests/tests/search-indexes.test.ts

```diff
@@ -41,6 +41,11 @@ const INDEX_DEFINITION = JSON.stringify({
   },
 });
 
+// The current timeout (2mins) is not enough for the search indexes to be created
+// and be queryable on Atlas. So we are increasing the timeout to 4mins.
+// This can not be more than 4mins as the mocha timeout is 4mins.
+const WAIT_TIMEOUT = 240_000;
```

Comment:
This is the same timeout as mocha so I worry that the mocha test timeout will be hit before the waitUntil below (since the test contains it and is this slower overall). So the test would still fail like it would if the waitUntil times out but then we'd get a worse error message because it wouldn't point at the wait line.